README.md

```diff
@@ -132,6 +132,15 @@ Options:
 
 ```
 
+#### Slack Notifications
+
+To enable Slack notifications during the deploy process create a [Slack Webhook](https://api.slack.com/incoming-webhooks) and add two entries `SLACK_WEBHOOK` and `SLACK_CHANNEL` to your `env.yml`.
+
+```
+SLACK_CHANNEL: '#devops'
```

Comment:
Quotes are needed because of the number sign. A number sign is the beginning of a comment in yaml.

We could auto add the number sign and then neither would be needed. 🤔